kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/FormServiceIntegrationTest.java

```diff
@@ -124,6 +128,28 @@ protected void runGetTaskFormTest( long processInstanceId ) {
         }
     }
 
+    @Test(expected = KieServicesException.class)
+    public void testGetTaskFormWithoutPermissioneViaUIClientTest() throws Exception {
+        changeUser(USER_YODA);
+        long processInstanceId = processClient.startProcess(CONTAINER_ID, HIRING_PROCESS_ID);
+        assertTrue(processInstanceId > 0);
+        try {
+            List<TaskSummary> tasks = taskClient.findTasksByStatusByProcessInstanceId(processInstanceId, null, 0, 10);
+            assertNotNull(tasks);
+            assertEquals(1, tasks.size());
+
+            Long taskId = tasks.get(0).getId();
+
+            changeUser(USER_JOHN);
+
+            uiServicesClient.getTaskForm(CONTAINER_ID, taskId, "en");
```

Comment:
Please check error message and HTTP code using [RestJmsSharedBaseIntegrationTest.assertClientException()](https://github.com/kiegroup/droolsjbpm-integration/blob/ec1e2110e5ff9942ef22cb346e94580347fbc65c/kie-server-parent/kie-server-tests/kie-server-integ-tests-common/src/main/java/org/kie/server/integrationtests/shared/basetests/RestJmsSharedBaseIntegrationTest.java#L147-L162) method .